server/tournaments/index.ts

```diff
@@ -30,6 +30,28 @@ function usersToNames(users: TournamentPlayer[]) {
 	return users.map(user => user.name);
 }
 
+export interface TournamentData {
```

Comment:
You haven't added this type to the return type of `getBracketData` yet.

Reply:
done!